code/modules/research/designs/mechfabricator_designs.dm

```diff
@@ -701,46 +692,55 @@
 	id = "borg_upgrade_defibrillator"
 	build_type = MECHFAB
 	build_path = /obj/item/borg/upgrade/defib
-	materials = list(MAT_METAL=15000, MAT_GLASS=15000, MAT_SILVER=10000, MAT_GOLD=10000, MAT_TITANIUM=5000, MAT_DIAMOND=5000)
-	construction_time = 120
+	materials = list(MAT_METAL = 8000, MAT_GLASS = 5000, MAT_SILVER = 4000, MAT_GOLD = 3000)
```

Comment:
Suitable since its vs having something else but primary question for my approval as a anti power gamer contributor does this need two or one slot on a borg?

Reply:
similar price to defib

Reply:
This is a better version since borgs don't have backpacks to sacrifice to hold it